socranop/data/man/socranop-ctl.1

```diff
@@ -0,0 +1,160 @@
+.\" ======================================================================
+.\"
+.\" The socranop-ctl(1) man page
+.\"
+.\" This man page has been (re)written adhering to the following
+.\" documentation: man(7), man-pages(7), tbl(1)
+.\"
+.\" ======================================================================
+.\"
+.TH SOCRANOP\-CTL 1 "2021\-07\-04" "${PACKAGE} ${VERSION}" "User commands"
+.\"
+.\" ======================================================================
+.\"
+.SH NAME
+socranop\-ctl \- control Soundcraft Notepad series mixer USB audio routing from command line
```

Comment:
I feel like we keep using similar but different phrases all over the place for the same kind of "short description" thing.  Should we standardize on one?

Reply:
Probably. For v0.5? Not sure.

With post-v0.5 ducker support, the programs will not be just about audio routing any more, and that will require new wording anyway.